app/controllers/assignments_controller.rb

```diff
@@ -2,6 +2,7 @@ class AssignmentsController < ApplicationController
     before_action :set_action_item, only:[:show, :edit]
     def index
         @assignments = authorize Assignment.all
+        @user = current_user
```

Comment:
check out my comment in the ```index.html.erb``` file but i don't think you need to define a var for current_user here.